### ISSUE TITLE
wireshark ,cookie pair

### ISSUE THREAD

**Issue by lyy06301** [FoxIO-LLC/ja4#134]
If there are multiple '=' in the cookie pair, the value will be truncated by the second '='. Should the complete value be extracted? For example, key1=value_key=value_value; The cookie extracted in ja4h is key1=value_key. Should it be key1=value_key=value_value, where key1 participates in sorting

**Comment by john-althouse** [comment]
That is correct. JA4H code should treat "x=y=z" as field "x" value "y=z". Does that make sense?
